specification/monitor/resource-manager/microsoft.insights/preview/2018-06-01-preview/guestDiagnosticSettingsAssociation_API.json

```diff
@@ -0,0 +1,372 @@
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "The request succeeded.",
+                        "schema": {
+                            "$ref": "#/definitions/GuestDiagnosticSettingsAssociationResource"
+                        }
+                    },
+                    "default": {
+                        "description": "An error occurred and the diagnostic settings association could not be retrieved.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            },
+            "delete": {
+                "description": "Delete guest diagnostics association settings.",
+                "tags": [
+                    "GuestDiagnosticsSettingsAssociation"
+                 ],
+                "operationId": "GuestDiagnosticsSettingsAssociation_Delete",                
+                "parameters": [
+                    {
+                        "name": "resourceUri",
+                        "in": "path",
+                        "required": true,
+                        "type": "string",
+                        "description": "The fully qualified ID of the resource, including the resource name and resource type.",
+                        "x-ms-skip-url-encoding": true
+                    },
+                    {
+                        "$ref": "#/parameters/DiagnosticSettingsAssociationNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "The diagnostic settings association was successfully deleted."
+                    },
+                    "204": {
+                        "description": "The diagnostic settings association does not exist. It may have already been deleted."
+                    },
+                    "default": {
+                        "description": "An error occurred and the diagnostic settings association could not be deleted.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/providers/microsoft.insights/guestDiagnosticSettingsAssociations": {
+            "get": {
+                "description": "Get a list of all guest diagnostic settings association in a subscription.",
+                "operationId": "guestDiagnosticsSettingsAssociation_ListBySubscriptionId",                
```

Comment:
This can be renamed to: `"guestDiagnosticsSettingsAssociation_List"` as the subscription ID is implicit.